java/src/org/openqa/selenium/ie/InternetExplorerDriver.java

```diff
@@ -173,6 +174,11 @@ public static RemoteWebDriverBuilder builder() {
     return RemoteWebDriver.builder().oneOf(new InternetExplorerOptions());
   }
 
+  @Beta
+  public static RemoteWebDriverBuilder builder(ClientConfig clientConfig) {
```

Comment:
I think this wouldn't work because there's an edit check in the base class builder that vets out for a combination and then throws an exception. We should probably avoid this and instead just resort to adding the overloaded constructors as we have done for the remaining webdriver implementations so that we are consistent across the board.

Reply:
Yes, we should follow the same proposed format as in the other driver classes.